.github/workflows/ci.yaml

```diff
@@ -148,7 +155,9 @@ jobs:
       - uses: Swatinem/rust-cache@v2
       - name: "Run tests"
         shell: bash
-        run: cargo nextest run --workspace --status-level skip --failure-output immediate-final --no-fail-fast -j 12
+        run: |
+          cargo nextest run --all-features --profile ci
+          cargo test --all-features --doc
```

Comment:
Why is this on the Windows tests? If anything, shouldn't it only be on the Linux tests, since doctests add so much overhead, and Windows is the bottleneck?

Reply:
`cargo insta` uses `nextest` to run the "normal" test but calls `cargo test --doc" for doctests. 

So the doctests run on both Windows and Linux. 

However, we never used the unreferenced-snapshots feature on Windows, so it felt useless to use `cargo insta` on Windows for a feature that we don't need. That's why it makes the call directly.

Reply:
Ah that makes sense, thanks! Doctests are so slow, honestly I'm tempted not to run them on Windows.